### DISCUSSION TITLE
求助铁铁，我是绳网新用户，该咋升级喵？

### DISCUSSION THREAD

**Discussion by littlehorseaaa** [share121/inter-knot#2135] [其他]
![11347cb6-ff16-43ef-a1d5-7b6e326789ab](https://github.com/user-attachments/assets/a028024c-da75-40e3-b633-595b4a55608e)

**Comment by jinyu07** [comment]
同问，有回答了，踢一下，谢谢

**Reply by cpu644** [reply]
你好！目前等级和你github公开仓库数量有关。

**Comment by JohnDoe-A** [comment]
同问喵

**Reply by cpu644** [reply]
你好！目前等级和你github公开仓库数量有关。

**Comment by nostalgiatan** [comment]
目前为止的升级机制就是开放公开仓库数量，每开放一个公开仓库等级就会加1。

**Comment by Sqrootof** [comment]
不是我说，这个github真的本身就很像绳网啊，尤其大陆还不能直接登录[哭笑不得_表情]
